code/modules/clothing/spacesuits/chronosuit.dm

```diff
@@ -167,6 +179,13 @@
 	else
 		finish_chronowalk(user, to_turf)
 
+/obj/item/clothing/suit/space/chronos/proc/create_phase_underlay(var/mob/user)
+	var/icon/user_icon = icon('icons/effects/alphacolors.dmi', "")
+	user_icon.AddAlphaMask(getFlatIcon(user))
```

Comment:
I'd love to avoid this but BYOND doesn't let me.

Here's a suggestion from @MrPerson a while ago which would allow this much more performantly: https://secure.byond.com/forum/?post=2065503
I don't want to be the one spam bumping it, so I call on _you_ to necro it once.

I'd love to hear of alternatives as well that accomplish the same goal 😄

Reply:
Gotcha covered fam

https://pastebin.com/e5aSsDB1

Doesn't work with visual contents or anything. I wrote this before visual_contents even existed.

Reply:
Can't you just keep this instead of generating this every move ?

Reply:
I'm gonna try to switch to appearances instead

Reply:
Haha color matrixes killed the underlay